modules/nw-routeadvertisements-example.adoc

```diff
@@ -0,0 +1,257 @@
+
+.An example of a `FRRConfiguration` CR generated by OVN-Kubernetes
+[source,yaml]
+----
+apiVersion: frrk8s.metallb.io/v1beta1
+kind: FRRConfiguration
+metadata:
+  name: ovnk-generated-abcdef
+  namespace: openshift-frr-k8s
+spec:
+  bgp:
+    routers:
+    - asn: 64512
+      vrf: blue
+      imports:
+      - vrf: default
+    - asn: 64512
+      neighbors:
+        - address: 172.18.0.5
+          asn: 64512
+          toReceive:
+            allowed:
+              mode: filtered
+          toAdvertise:
+            allowed:
+              prefixes:
+              - 10.0.1.0/24
+      prefixes:
+      - 10.0.1.0/24
+      imports:
+      - vrf: blue
+  nodeSelector:
+    matchLabels:
+      kubernetes.io/hostname: ovn-worker
+----
+
+The generated `FRRConfiguration` object configures the subnet `10.0.1.0/24`, which belongs to network blue, to be imported into the default VRF and advertised to the `172.18.0.5` neighbor. An `FRRConfiguration` object is generated for each network and nodes selected by a `RouteAdvertisements` CR with the appropriate prefixes that apply to each node.
+
+When the `targetVRF` field is omitted, the routes are leaked and advertised over the default VRF. Additionally, routes that were imported to the default VRF after the definition of the initial FRRConfiguration object are also imported into the blue VRF.
+
+[id="advertising-pod-ips-from-a-user-defined-network-over-bgp-with-vpn_{context}"]
+== Advertising pod IPs from a cluster user-defined network over BGP with VPN
+
+In this scenario, a VLAN interface is attached to the VRF device associated with the blue network. This setup provides a _VRF lite_ design, where FRR-K8S is used to advertise the blue network only over the corresponding BGP session on the blue network VRF/VLAN link to the next hop Provide Edge (PE) router. The red tenant uses the same configuration. The blue and red networks are labeled as `export: true`. The following diagram illustrates this configuration:
+
+image::524-openshift-bgp-ovn-k8s-vrf-lite-0325.png[Advertising pod IPs from a cluster user-defined network over BGP with VPN]
+
+Red CUDN::
+- A VRF named `red` associated with a CUDN named `red`
+- A VLAN interface attached to the VRF device and connected to the external PE router
+- An assigned subnet of `10.0.2.0/24`
+
+Blue CUDN::
+- A VRF named `blue` associated with a CUDN named `blue`
+- A VLAN interface attached to the VRF device and connected to the external PE router
+- An assigned subnet of `10.0.1.0/24`
+
+[NOTE]
+====
+This approach is available only when you use OVN-Kubernetes in local gateway mode by setting `routingViaHost=true` and does not support EgressIP advertisements.
```

Comment:
I have been imprecise with my comment here: https://github.com/openshift/openshift-docs/pull/89983#discussion_r2215739963

It is more of Egress IPs are not supported at all in this configuration. Not only advertisements, but the plain use of Egress IPs.